packaging/wix/LICENSE.rtf.in

```diff
@@ -1,7 +1,7 @@
 {\rtf1\ansi\ansicpg1252\deff0\nouicompat{\fonttbl{\f0\fnil\fcharset0 Courier New;}}
 {\colortbl ;\red0\green0\blue255;}
 {\*\generator Riched20 10.0.14393}\viewkind4\uc1
-\pard\qj\ul\b\f0\fs22\lang1036 Mixxx version 2.4, Digital DJ'ing software.\ulnone\b0\fs24\par
+\pard\qj\ul\b\f0\fs22\lang1036 Mixxx @CMAKE_PROJECT_VERSION@, Digital DJ'ing software.\ulnone\b0\fs24\par
 \fs22 Copyright (C) 2001-2023 Mixxx Development Team\par

```

Comment:
Should we have an automatic copyright year as well?

Reply:
While this is a good idea, I like to postpone the work for now. This is not trivial, because the LICENSE.rtf is configured at configure time, but I think we need the time from the HEAD commit which may change without reconfigure. On the other hand, I don't want to force reconfigure the build after every commit.